extensions/typescript-language-features/src/languageFeatures/fileConfigurationManager.ts

```diff
@@ -185,6 +185,8 @@ export default class FileConfigurationManager extends Disposable {
 			generateReturnInDocTemplate: config.get<boolean>('suggest.jsdoc.generateReturns', true),
 			includeCompletionsForImportStatements: config.get<boolean>('suggest.includeCompletionsForImportStatements', true),
 			includeCompletionsWithSnippetText: config.get<boolean>('suggest.includeCompletionsWithSnippetText', true),
+			// @ts-expect-error until 4.4
+			allowIncompleteCompletions: config.get<boolean>('suggest.allowIncompleteCompletions', true),
```

Comment:
Following up on our discussion earlier today:  we agreed that this should not be user configurable